interface/src/app/gene/search/directives.js

```diff
@@ -0,0 +1,321 @@
+angular.module('adage.gene.search.directives', [
```

Comment:
By the way, when all these files are in the same module, you can even divide **directives.js** further if you want.